labcore/analysis/hvapps.py

```diff
@@ -391,6 +400,7 @@ async def load_and_preprocess(self, *events: param.parameterized.Event) -> None:
             self.data_out = data
             t1 = datetime.now()
             self.info_label.value = f"Loaded data at {t1.strftime('%Y-%m-%d %H:%M:%S')} (in {(t1-t0).microseconds*1e-3:.0f} ms)."
+        return
```

Comment:
There is no need for this return, it is implicit if you are not returning anything and the function ends